contact.html

```diff
@@ -4,59 +4,136 @@
+        <div class="align-center contact-buttons">
+          <a class="button light-button" id="show-adv-form" href="#">Request ADV and Privacy Policy</a>
+          <a class="button sub-button light-sub-button" id="show-webinar-form" href="#">Webinar Request</a>
+        </div>
+      </div>
     </main>
-    <section class="wrapper">
 
-    </section>
-    <footer>
+    <section class="section-contact">
       <div class="wrapper">
-        <div class="leftfooter align-left clearfix:after">
-          <img src="images/logo_white.png" class="footer-logo">
-          <p>1886 Metro Center Dr. Suite 620 Reston, VA 20190<br>
-          Phone (703) 757-0419  -  Fax (703) 757-0430<br>
-          For more information please email us at: <a href="mailto:tjdoremus@prestonwealthadvisors.com">tjdoremus@prestonwealthadvisors.com</a> or <a href="mailto:gmattingly@prestonwealthadvisors.com">gmattingly@prestonwealthadvisors.com</a></p>
-        </div>
-      <div class="centerfooter align-left clearfix:after">
-        <h3>Find us here!</h3>
-        <iframe src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d3102.822002430413!2d-77.34076030000001!3d38.9508946!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x89b6482e2c339799%3A0xcf727a5bdcbdbc6c!2s1886+Metro+Center+Dr+%23620%2C+Reston%2C+VA+20190!5e0!3m2!1sen!2sus!4v1438570536650" width="400" height="300" allowfullscreen></iframe>
-      </div>
-      <div class="rightfooter align-right">
-        <h3>Disclaimer</h3>
-        <p>It should not be assumed that recommendations made in the future will be profitable or will equal the performance of the securities in this list. All performance calculations prior to 2013 have been done by the research department and have not been audited or verified by PWA or any outside 3rd party. The performance was calculated using live models and actual trades but not through PWA's platform. Past performance may not be indicative of future results and the performance of a specific individual client account may vary substantially from the composite performance results. Therefore, no current or prospective client should assume that future performance will be profitable, or equal either the Preston Wealth Advisors’ composite performance results reflected above, or the performance results for any of the comparative index benchmarks provided.</p>
+        <div class="section-column-contact align-left">
+          <div class="adv-request" id="adv-form">
+          <h2 class="form-heading">Request for ADV and Privacy Policy</h2>
+          <p class="section-p-contact">Call (703) 757-0419 or use the form below to learn more about how we can help you.</p>
+
+          <form>
+            <div class="adv-form">
+              <fieldset>
+                <div class="form-row">
+                  <div class="name-container">
+                    <input type="text" name="name[first]" placeholder="First Name (Required)" id="name" class="name-field">
```

Comment:
I've never seen a name value written this way? It's valid from what I've tested, but it's a bit bizarre since the ['string'] are typically saved for arrays.